### PR TITLE
Added some steps in the workflow tutorial

### PR DESCRIPTION
**Pull request by teoEnache**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

Describe the scope of this pull request. Is this a new feature or a bug fix?

### Checklist

- [ ] Checked spelling and grammar
- [ ] Provided code samples where relevant
- [ ] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/getting-started/first-workflow.rst [hunk 1]
```diff
@@ -4,106 +4,208 @@
...
+
+.. figure:: _assets/step01_signUp_2of3.png
+   :align: center
+   :alt: SignUp2
+
+.. figure:: _assets/step02_signUp_3of3.png
+   :align: center
+   :alt: SignUp3
+
+Congratulations, you successfully created an UP42 account! In order to get access to the free 10 000 UP42 credits or the equivalent of 100 Euro/Dollars, you need to provide the credit card details. You will not be charged from this credit card.
```
Comment (thaisbendixen) [inline source/getting-started/first-workflow.rst:42]: small suggestion:
"you need to provide your credit card details" or "you need to provide credit card details"

File: source/getting-started/first-workflow.rst [hunk 2]
```diff
@@ -4,106 +4,208 @@
...
-A workflow consists of *Data blocks* and *Processing blocks*. A workflow always start with a data
-block. Its output can then be used by a processing block, or a chain of processing blocks.
+In the UP42 platform, a workflow consists of *data blocks* and *processing blocks*. The first block is always a data block. This data block can be followed by one or more processing blocks.
 
-You should now see the workflow editor. Under the "Data" section,
-click on **Add Block** and select the "SPOT 6/7 Streaming" block
-to view its details. Then click **Add Block** at the bottom of the block
-description. Your workflow will automatically be saved whenever you add or
-remove a block.
+In this example, a workflow based on the detection of ships from high-resolution SPOT images (1.5 m spatial resolution) will be shown. The first step is to select the data block from which these ships will be extracted. Click on *Add data*.
```
Comment (thaisbendixen) [inline source/getting-started/first-workflow.rst:74]: "The first step is to select the data block from which ships will be extracted." i would remove "these"

**Review by thaisbendixen — APPROVED** [review]
Amazing!!! Only a few small suggestions, mostly style

File: source/getting-started/first-workflow.rst (hunk 1 above)
Comment (teoEnache) [reply source/getting-started/first-workflow.rst:42]: roger that

File: source/getting-started/first-workflow.rst (hunk 2 above)
Comment (teoEnache) [reply source/getting-started/first-workflow.rst:74]: roger that

**Pull request merged**